.pre-commit-config.yaml

```diff
@@ -7,6 +7,11 @@
 # default_language_version:
 #   python: python3
 repos:
+  - repo: https://github.com/psf/black
+    rev: 23.12.1
+    hooks:
+      - id: black
+
```

Comment:
Please consider `ruff-format` instead.

I think the examples should be left alone for now too. 

If you are planning on an auto-formatting commit, please also add the Ruff `COM` rule there, and make sure not to squash-commit the PR including it.

Reply:
Added formatting in #793 with ruff-format